src/chem/modis_aod2ioda.py

```diff
@@ -0,0 +1,183 @@
+            over_land = np.logical_not(land_sea_flag == 0)
+            UNC = np.where(over_land, unc_land, np.add(0.05, np.multiply(0.15, aod)))
+            UNC_2 = UNC[0]
+
+            if first:
+                self.loc_mdata['latitude'] = lats
+                self.loc_mdata['longitude'] = lons
+                self.loc_mdata['datetime'] = self.writer.FillNcVector(obs_time, "datetime")
+                self.loc_mdata['latitude'] = np.concatenate((self.loc_mdata['latitude'], lats[0:1]))
+                self.loc_mdata['longitude'] = np.concatenate((self.loc_mdata['longitude'], lons[0:1]))
+                self.loc_mdata['datetime'] = np.concatenate((self.loc_mdata['datetime'],
+                                                            self.writer.FillNcVector(obs_time[0:1], "datetime")))
+            else:
+                self.loc_mdata['latitude'] = np.concatenate((self.loc_mdata['latitude'], lats))
+                self.loc_mdata['longitude'] = np.concatenate((self.loc_mdata['longitude'], lons))
+                self.loc_mdata['datetime'] = np.concatenate((self.loc_mdata['datetime'],
+                                                            self.writer.FillNcVector(obs_time, "datetime")))
+
+            for ncvar, iodavar in obsvars.items():
+                data = aod.astype('float32')
+                err = UNC
+                err_2 = err[0]
+                if first:
+                    self.outdata[self.varDict[iodavar]['valKey']] = aod
+                    self.outdata[self.varDict[iodavar]['errKey']] = err
+                    self.outdata[self.varDict[iodavar]['qcKey']] = QC_flag
+                    self.outdata[self.varDict[iodavar]['valKey']] = np.concatenate(
+                        (self.outdata[self.varDict[iodavar]['valKey']], data[0:1]))
+                    self.outdata[self.varDict[iodavar]['errKey']] = np.concatenate(
+                        (self.outdata[self.varDict[iodavar]['errKey']], err[0:1]))
+                    self.outdata[self.varDict[iodavar]['qcKey']] = np.concatenate(
+                        (self.outdata[self.varDict[iodavar]['qcKey']], QC_flag[0:1]))
+
+                else:
+                    self.outdata[self.varDict[iodavar]['valKey']] = np.concatenate(
+                        (self.outdata[self.varDict[iodavar]['valKey']], data))
+                    self.outdata[self.varDict[iodavar]['errKey']] = np.concatenate(
+                        (self.outdata[self.varDict[iodavar]['errKey']], err))
+                    self.outdata[self.varDict[iodavar]['qcKey']] = np.concatenate(
+                        (self.outdata[self.varDict[iodavar]['qcKey']], QC_flag))
+
+            first = False
+        self.writer._nvars = len(obsvars)
+        self.writer._nlocs = len(self.loc_mdata['longitude'])
+
+
+def main():
+
+    # get command line arguments
+    # Usage: python blah.py -i /path/to/obs/2021060801.nc /path/to/obs/2021060802.nc ... /path/to/obs/2021060823.nc
```

Comment:
you don't have the -o output info here in the usage.

Reply:
I will add this.